decidim-courses/app/models/decidim/course.rb

```diff
@@ -42,10 +42,7 @@ class Course < ApplicationRecord
              class_name: "Decidim::Courses::RegistrationType",
              dependent: :destroy
 
-    validates_upload :hero_image
     mount_uploader :hero_image, Decidim::HeroImageUploader
-
-    validates_upload :banner_image
```

Comment:
why are this validations outdated?

Reply:
They stopped using them in Decidim after adding
`mount_uploader :hero_image, Decidim::HeroImageUploader`
But I'll leave them anyways since our tests need them.